README.md

```diff
@@ -2229,6 +2229,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Yash Khokhaneshiya](https://github.com/ykhokhaneshiya)
   - [Yash Patel](https://github.com/yash-0025)
   - [Yash Pimple](https://github.com/YashPimple)
+  - [Yashpreet Singh](https://github.com/yashpreet2707)
```

Comment:
Please add your name in line 2239, after Yashesvinee V.
Let us know if you need any help with that. 🙂

Reply:
yup sorry, I've closed this PR and will create a new one. Thanks a lot 🤝